README.md

```diff
@@ -102,15 +121,18 @@ The client will run a chain with a single validator (Alice) and start producing
 2024-03-28 11:49:18 ✨ Imported #2 (0x8226…df51)
 2024-03-28 11:49:19 💤 Idle (0 peers), best: #2 (0x8226…df51), finalized #1 (0x89e3…c6c4), ⬇ 0 ⬆ 0
 ```
-## Documentation
-
-The official documentation is available at [docs.zkverify.io](https://docs.zkverify.io/).
 
 ## Docker
 
 zkVerify includes some Docker files for building the client and running one or more nodes locally.
 For more information, see [docker/README.md](docker/README.md).
 
+We also provide Docker images to run a validator, boot or RPC node directly in the public testnet, via a user-friendly installation and deployment process. Please take a look at [How to run a node](https://docs.zkverify.io/tutorials/how_to_run_a_node/getting_started) section of the official documentation. 
+
+## Documentation
+
+The official documentation is available at [docs.zkverify.io](https://docs.zkverify.io/).
+
```

Comment:
Maybe add here reference to official Discord

Reply:
Doesn't seem the correct place here, besides it's referenced in the docs